src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -7768,17 +7768,16 @@ HRESULT DacStackReferenceWalker::Next(ULONG count, DacGcReference stackRefs[], U
     {
         stackRefs[i].dwType = CorReferenceStack;
         stackRefs[i].vmDomain.SetDacTargetPtr(domain);
+        stackRefs[i].i64ExtraData = 0;
 
         const SOSStackRefData &sosStackRef = mList.Get(i);
         if (sosStackRef.Flags & GC_CALL_INTERIOR)
         {
-            stackRefs[i].i64ExtraData = GC_CALL_INTERIOR;
-            stackRefs[i].objHnd.SetDacTargetPtr(CLRDATA_ADDRESS_TO_TADDR(sosStackRef.Address));
+            stackRefs[i].pObject = CLRDATA_ADDRESS_TO_TADDR(sosStackRef.Object) | 1;
```

Comment:
So i64ExtraData no longer needs to be set to GC_CALL_INTERIOR in this branch?

Reply:
The original behavior was that `i64ExtraData` was not set at all for interior pointers.  (It probably _should_ be set, but it's not.)  Setting it to 0 in both cases just makes sure we preserve the previous behavior that VS might have taken a dependency on.

I'm solving this a different way in the upcoming change to improve performance.  (That PR will come later, after VS takes a look at it.)

Reply:
Sounds good.